radar/platforms-and-services/azure-database-mysql.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Perhaps we can delete this section as not used anymore? CSP doesn't allow creation of new instances neither